roles/redpanda_broker/templates/console/defaults.j2

```diff
@@ -0,0 +1,41 @@
+{
+  "kafka": {
```

Comment:
Is this deployed without SASL authentication by default?

Reply:
Yes, as I haven't spent the time to get SASL enabled in mainline RP yet. Custom config is available to enable it if desired by the end user.